pkg/server/server_test.go

```diff
@@ -0,0 +1,213 @@
+package server
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"strings"
+	"testing"
+
+	authenticationv1beta1 "k8s.io/api/authentication/v1beta1"
+
+	"github.com/heptio/authenticator/pkg/config"
+	"github.com/heptio/authenticator/pkg/token"
+)
+
+func verifyBodyContains(t *testing.T, resp *httptest.ResponseRecorder, s string) {
```

Comment:
Nit: this could have call to [`t.Helper()`](https://golang.org/pkg/testing/#T.Helper) to make nicer test failure messages.

Reply:
Done